composer.json

```diff
@@ -178,24 +178,38 @@
+                "2927347 - Having Entity Browser 2.x on the codebase breaks upgrade path": "https://www.drupal.org/files/issues/2019-01-10/Having-Entity-Browser-breaks-upgrade-path-2927347-14.patch"
             },
             "drupal/entity_embed": {
                 "2511404 - Image entities/fields embedded using Entity Embed cannot be linked in CKEditor": "https://www.drupal.org/files/issues/entity_embed_links-2511404-31.patch"
             },
             "drupal/plugin": {
                 "2647312 - Use SubFormState in plugin selectors": "https://www.drupal.org/files/issues/plugin_2647312_20.patch"
             },
+            "drupal/views_field_formatter": {
+                "2980093 - Can't uninstall module": "https://gist.githubusercontent.com/AndreyMaximov/fdf9128022266188cbce2cc82be3d099/raw/7797fd965915de32820e04fcb4c8189ac427fb62/2980093_viewsfieldformatter_get_on_null.patch"
+            },
             "drupal/views_data_export": {
                 "2789531 - Support for batch operations": "https://www.drupal.org/files/issues/2789531-105.patch"
             },
-            "drupal/views_field_formatter": {
-                "2874811 - Add option to show/hide empty views": "https://www.drupal.org/files/issues/Option-added-to-hide-empty-views-2874811.patch"
+            "drupal/easy_breadcrumb": {
+                "2971408 - Easy Breadcrumb could not determine the title to use for /node/{node}/translations": "https://www.drupal.org/files/issues/2018-07-13/2971408_node_edit-5.patch"
             },
             "drupal/core": {
                 "1236098 - Notice: Undefined index in _color_rewrite_stylesheet()": "https://www.drupal.org/files/issues/undefined-index-in-_color_rewrite_stylesheet-1236098-37.patch",
                 "2484693 - Telephone Link field formatter InvalidArgumentException with 5 digits or fewer in the number": "https://www.drupal.org/files/issues/2484693-54.patch",
                 "2862702 - PrepareModulesEntityUninstallForm::formTitle does not exist": "https://www.drupal.org/files/issues/2862702-3.patch"
+            },
+            "drupal/crop": {
+                "3025005 - The Crop requirements is NOK is media_entity 8.x-2.0 installed via composer": "https://www.drupal.org/files/issues/2019-01-10/requirements-is-NOK-3025005-2.patch"
+            },
+            "drupal/media_entity": {
+                "2918172 - Media Entity upgrade -> add revision fields": "https://www.drupal.org/files/issues/2918172-5.patch",
+                "2918172 - Media Entity upgrade -> core fails on absent column revision_uid": "https://www.drupal.org/files/issues/2918172-6-8.4.4.patch"
+            },
+            "drupal/blazy": {
+                "Remove core media dependencies to unblock upgrade": "https://raw.githubusercontent.com/afi13/openy/OPENY-231/patches/blazy/remove-core-media-dependencies.patch"
```

Comment:
this should be on drupal.org , please

Reply:
I think it doesn't make sense. This is a hack for Open Y upgrade path only which will be removed in the next release.
The problem here, that according to documentation we should upgrade code base to Drupal 8.6.x, run db updates and only then upgrade code of contrib modules. But we can't use this approach, so we are upgrading Drupal core and contrib modules together. And Blazy module defines service with dependency to Drupal core media, which is not enabled before db updates. And any drush command fails because of that.
That's why we need this patch which is not useful for people on drupal.org.